### DISCUSSION TITLE
creating repository is restricted

### DISCUSSION THREAD

**Discussion by abdulbasithp** [community/community#32735] [Other Feature Feedback, Questions, & Ideas]
The repository abdulbasithp has been retired and cannot be reused.
i can't create repository as same as my username

**Comment by radar-blip** [comment]
Yes you can, try it again. Maybe you spelled it wrong.

**Reply by abdulbasithp** [reply]
no chance tried again 
"The repository abdulbasithp has been retired and cannot be reused."
same output what will do

**Comment by airtower-luna** [comment]
Did you recently change your username? That message is part of a mechanism meant to protect against hostile takeovers of repository URLs, but I've seen it get in the way of people who changed their usernames and then changed them back. In that case you'd need to [contact GitHub support](https://support.github.com/request).
